srcpkgs/python3-reedsolo/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-reedsolo'
+pkgname=python3-reedsolo
```

Comment:
Shouldn't it be reedsolomon?

Reply:
It seems the python package calls itself `reedsolo` (https://pypi.org/project/reedsolo/). Though their GitHub repo is called reedsolomon.